gesturelibrary/include/gesturelib.h

```diff
@@ -59,6 +62,15 @@ void init_gesturelib();
 /// @return
 int process_touch_event(touch_event_t* touch_event, gesture_event_t* gestures, int max_gestures);
 
+/// @brief assign a group ID to this touch_event. Update group_heads to reflect this. If this is not a DOWN event
+///        and there are no groups being tracked, do not assign an ID. If this is a DOWN event but MAX_TOUCHES groups
+///        are already being tracked, do not assign an ID.
+/// @param touch_event
+/// @return
+unsigned int assign_group(touch_event_t* touch_event);
+
+float squared_distance(touch_event_t* a, touch_event_t* b);
```

Comment:
I don't think this makes sense to expose as part of the gesturelib API.
Please declare it static in gesturelib.c

Reply:
Alternatively, make a utils folder and place it there.

Reply:
that makes sense. I'll make it private in the c file